modules/products/products.resolver.js

```diff
@@ -0,0 +1,17 @@
+const productsService = require('./products.service');
+
+const productsQuery = {
+  getAllProducts: () => productsService.getAllProducts(),
+  getProductsById: (parent, args) => productsService.getProductsById(args.id),
+  getProductsByOptions:  (parent, args) => productsService.getProductsByOptions(args) 
```

Comment:
Please, merge `getProductsByOptions` and `getAllProducts` into one method that will take filter but if the filter is not provided will use default one

Reply:
merged